packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -113,37 +112,14 @@ export default class SalesforceCRMService implements CRM {
     });
   };
 
-  private salesforceContactCreate = async (attendees: Person[]) => {
+  private getSalesforceUserFromEmail = async (email: string) => {
```

Comment:
Get the Salesfoce user info from the email provided. Assuming that the email on Cal.com is the same as Salesforce